### PR TITLE
Create example page for team content and add dummy profiles

### PR DESCRIPTION
**Pull request by Piumal1999**
Ex: Team page, Alumni page,IRC Researchers etc.

## Security checks
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? YES

## Related PRs
PR #256

### CODE REVIEW

File: src/main/webapp/examples/team.html [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+<main>
+    <section>
+        <div class="container text-center">
+            <div class="mb-5 text-left h2">
+                <span>Lorem ipsum dolor sit amet</span>
+            </div>
+
+            <div id="teamContent"></div>
+
+            <button id="btn" class="btn btn-link btn-outline-white btn-info">SEE MORE</button>
```
Comment (jayasanka-sack) [inline src/main/webapp/examples/team.html]: Change button id to a unique one.  'btn' might be repeated when using with other pages.

File: src/main/webapp/examples/team.html [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+                </div>
+            </div>
+        </div>
+        {{/data}}
+    </div>
+</script>
+
+<!--Script for load team data-->
+<script>
+    $.ajax({
```
Comment (jayasanka-sack) [inline src/main/webapp/examples/team.html]: wrap this with a function. call it from line 139.

File: src/main/webapp/examples/team.html [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        success: function (data) {
+
+            //slice array to two parts
+            if(data.length>=8){
+                let partOne = data.slice(0, 8);
+                let partTwo = data.slice(8);
+
+                //mustache render
+                let contentPartOne = Mustache.render($("#templateTeam").html(), {"data": partOne});
+                let contentPartTwo = Mustache.render($("#templateTeam").html(), {"data": partTwo});
```
Comment (jayasanka-sack) [inline src/main/webapp/examples/team.html]: Do not render it here. Render only when someone clicks the button.

File: src/main/webapp/examples/team.html [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+                //display first 8 profiles
+                $("#teamContent").html(contentPartOne);
+
+                //hide button
+                $("#btn").click(function () {
+                    $("#teamContent").append(contentPartTwo);
+                    document.getElementById("btn").style.display = "none";
```
Comment (jayasanka-sack) [inline src/main/webapp/examples/team.html]: You can use jQuery selector here: $("#btn").style.display = none;

File: src/main/webapp/examples/team.html (hunk 1 above)
Comment (Piumal1999) [reply src/main/webapp/examples/team.html]: Done

File: src/main/webapp/examples/team.html (hunk 3 above)
Comment (Piumal1999) [reply src/main/webapp/examples/team.html]: Done

File: src/main/webapp/examples/team.html (hunk 4 above)
Comment (Piumal1999) [reply src/main/webapp/examples/team.html]: 🤩

File: src/main/webapp/examples/team.html (hunk 2 above)
Comment (Piumal1999) [reply src/main/webapp/examples/team.html]: Done

File: src/main/webapp/examples/team.html [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+                    $("#btnShowMore").style.display = "none";
+                }
+            }
+        });
+    }
+
+</script>
+
+<!-- Script for static navbar and footer -->
+<script src="../assets/js/navigation.js"></script>
```
Comment (jayasanka-sack) [inline src/main/webapp/examples/team.html]: Move this to line number 60

File: src/main/webapp/examples/team.html [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+                </div>
+            </div>
+        </div>
+        {{/data}}
+    </div>
+</script>
+
+<!--Script for load team data-->
+<script>
+    function loadProfiles() {
```
Comment (jayasanka-sack) [inline src/main/webapp/examples/team.html:103]: Add this to line 142. We can use the same script tag.

**Comment by jayasanka-sack** [comment]
Thanks @Piumal1999 !

**Comment by jayasanka-sack** [comment]
#273

**Pull request merged**